src/lib/compress.c

```diff
@@ -75,6 +75,7 @@ __RCSID("$NetBSD: compress.c,v 1.23 2012/03/05 02:20:18 christos Exp $");
 #include <string.h>
 #include <stdlib.h>
 
+#include "../common/constants.h"
```

Comment:
That's what I tried to say - we should put includes into single rnp folder. Or merge into single one. All advices how do this better are welcome, actually this is the main goal of reviewing PRs, isn't it?